browser.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Easier to just add `/* eslint-globals navigator */` than the XO overrides in package.json.

Reply:
Fixed!